src/clj/harja/palvelin/integraatiot/api/yllapitokohteet.clj

```diff
@@ -135,10 +135,13 @@
         alikohteet))
 
 (defn paivita-yllapitokohde [vkm db kayttaja {:keys [urakka-id kohde-id]} data]
-  (log/debug (format "Päivitetään urakan (id: %s) kohteelle (id: %s) tiedot käyttäjän: %s toimesta"
-                     urakka-id
-                     kohde-id
-                     kayttaja))
+  ;; Oletamme että rajapintaa ei käytetä. Jos ao. varoitusviesti ilmestyy Graylogiin, voidaan ohjata
+  ;; ensin käyttäjä tekemään tarvittava muutos käyttöliittymältä. Sitten on vielä tehtävä VKM-rajapinnan muuttumiseen liittyviä tehtäviä VHAR-5246 mukaisesti
+  (log/warn "Rajapintaa paivita-yllapitokohde kutsuttiin.")
```

Comment:
log/info maybe?

Reply:
Tarkoituksella varoitus, jotta varmasti huomaamme, jos näitä kutsutaan. Jos näitä kutsutaan, meidän pitää tehdä paljon koodimuutoksia näihin, joten on hyvin tärkeää että nämä lokit huomattaisiin.